python/README.md

```diff
@@ -0,0 +1,37 @@
+# Build python as static PIE
+
+Build the [python](https://www.python.org/) interpreter as a static PIE ELF running on Linux.
+
+## Requirements
+
+Make sure the following packages are installed:
+* gcc
+* GNU Make
+
+For Debian based systems, run the command below to install all requirements:
+```
+$ sudo apt install build-essential
+```
+
+## Build
+
+The `python` static PIE ELF file is located in the current folder.
+If you want to rebuild it, run:
+
+```
+$ ./build.sh
+```
+
+## Running
+
+To run the `python` binary, you need to specifiy where it can find the `Lib`
+folder. This can be done by setting the `PYTHONPATH` variable. The `Lib` folder
+is normally found in the source files. Here, there is also in the `rootfs`
+folder. Therefore, to run the binary, you can do this:
```

Comment:
```suggestion
To run the `python` binary, you need to specify where it can find the `Lib/` folder.
This can be done by setting the `PYTHONPATH` variable.
The `Lib/` folder is normally found in the source files.
Here, there is also in the `rootfs/` folder.
Therefore, to run the binary, use this:
```